security/pkg/pki/ca/ca.go

```diff
@@ -235,3 +243,10 @@ func BuildSecret(saName, scrtName, namespace string, certChain, privateKey, root
 		Type: secretType,
 	}
 }
+
+func updateCertInConfigmap(namespace string, client corev1.CoreV1Interface, keyCertBundle util.KeyCertBundle) error {
+	cert, _, _, _ := keyCertBundle.GetAllPem()
```

Comment:
could you pls explain what's difference between ```bundle.certBytes``` and ``` bundle.rootCertBytes```(4th output param), could we just use ``` bundle.rootCertBytes``` ?

Reply:
certBytes is the signing cert bytes. rootCertBytes is the root cert bytes.
Yes, I think we should use root cert here, so that the root of trust for Citadel agent doesn't need to be updated if the signing cert is rotated.